### ISSUE TITLE
[FEATURE] Custom Chart with timestamp format

### ISSUE THREAD

**Issue by unocelli** [frangoteam/FUXA#1470]
### Discussed in https://github.com/frangoteam/FUXA/discussions/1458

<div type='discussions-op-text'>

<sup>Originally posted by **timmpo** October 27, 2024</sup>
i have tried to send coustom data to line chart, but have trouble to get time series axis to work,
when paramLines return x axis in unix-time (seconds) it plots the data but need some conversion to human readable format, i have tried to send timestamps in string format etc (2024/10/10-17:50) but it dont works.
what is the correct time format to send ? 

working example of paramLines returning values, but need some human readable conversion:

```typescript
 {
    id: 't_b070199c-92384f8b',
    name: 'chartDataPoint',
    device: 'FUXA Server',
    color: '#4484ef',
    label: 'chartDataPoint',
    yaxis: 1,
    x: [
      1730022583, 1730022593,
      1730022603, 1730022613,
      1730022623, 1730022633,
      1730022643, 1730022653,
      1730022663, 1730022673,
      1730022683, 1730022693,
      1730022703, 1730022713,
      1730022723, 1730022733,
      1730022743, 1730022753
    ],
    y: [
      93, 93, 78, 57, 37, 52, 49,
      16, 11, 35, 98, 39, 91, 55,
      53, 60, 66, 51
    ]
  }
]
``` </div>

**Comment by unocelli** [comment]
I'm working on this feature

**Comment by unocelli** [comment]
Available in master branch

**Comment by unocelli** [comment]
I’m going to close this as resolved. let me know if you have any issues.

**Comment by hesitatenot** [comment]
Hello @unocelli , 
The project [custom_chart_datetime.json](https://github.com/user-attachments/files/17894437/custom_chart_datetime.json)  works well for the basic version; however, the project, especially the `$invokeObject('chart', 'setValues', values);`  line does not work for the Fuxa-Pro version(docker fuxa-pro:1.2.6). Here's the error message. Could you please have a look?

<img width="442" height="84" alt="Image" src="https://github.com/user-attachments/assets/828feceb-a810-48aa-b368-ab52c9a6254d" />

**Comment by unocelli** [comment]
@hesitatenot You can try the version  fuxa-pro:latest (pre release), should be fixed in the next version 1.2.7 that is coming
